analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TestAnalysisComparer_2.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
Why is this actually `_2`? To prove that it works on multiple files? 
I see the `flows` here as a common topic. Can this be `TestAnalysisComparer.Flows.cs` instead? Or `TestAnalysisComparer.Secondary.cs` to play with all secondary locations?

The first one could be `TestAnalysisComparer.Primary.cs` then.